python/pyarrow/orc.py

```diff
@@ -117,21 +185,92 @@ def read(self, columns=None):
         return self.reader.read(columns=columns)
 
 
+_orc_writer_args_docs = """file_version : {"0.11", "0.12"}, default "0.12"
+    Determine which ORC file version to use. Hive 0.11 / ORC v0 is the older
+    version as defined `here <https://orc.apache.org/specification/ORCv0/>`
+    while Hive 0.12 / ORC v1 is the newer one as defined
+    `here <https://orc.apache.org/specification/ORCv1/>`.
```

Comment:
Beware the reST syntax for hyperlinks: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases

```suggestion
    version as defined `here <https://orc.apache.org/specification/ORCv0/>`_
    while Hive 0.12 / ORC v1 is the newer one as defined
    `here <https://orc.apache.org/specification/ORCv1/>`_.
```

Reply:
Really thanks! Fixed.